multiversx_sdk_wallet/interfaces.py

```diff
@@ -11,4 +11,4 @@ class IRandomness(Protocol):
 
 
 class IAddress(Protocol):
-    def bech32(self) -> str: ...
+    def to_bech32(self) -> str: ...
```

Comment:
Breaking change, so the new wallet isn't compatible with the old core anymore.

However, if we postpone this, and leave `bech32` (only here in the interface), the tests still pass, correct?